jenkins-pipelines/alternator-rolling-upgrade.jenkinsfile

```diff
@@ -0,0 +1,16 @@
+#!groovy
```

Comment:
please rename this file to `rolling-upgrade-centos7-alternator.jenkinsfile`
so it would automatically can be picked into the correct folder in the next branch we create.

Reply:
Done